src/sentry/debug_files/utils.py

```diff
@@ -0,0 +1,13 @@
+import sentry_sdk
+
+
+def size_in_mb(string: str) -> int:
```

Comment:
no need to round to MB, just output bytes.

datadog can automatically show you the right unit if you manually edit the metric metadata btw.